daemon/core.h

```diff
@@ -146,6 +146,7 @@ class Core : public QThread, public LogTarget
     void grabShortcut(const uint &timeout, QString &shortcut, bool &failed, bool &cancelled, bool &timedout, const QDBusMessage &message);
     void cancelShortcutGrab();
 
+    void on_shortcut(const Ids& ids) const;
```

Comment:
Is there a reason for not following the naming style here?

Reply:
Because Qt vs STL signal/slot cardinality. :smiley_cat: (Actually `onShortcutGrabbed` signal is the "black sheep" in this context.)
However it doesn't matter, just feels weird seeing a Qt signal named "onXYZ".